### PR TITLE
feat: add page component

### PR DESCRIPTION
**Pull request by ryuuzake**
Here are some proves that the page component works

| Storybook                                                                                 | Playground                                                                                |
|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
| image | image |
| image | image |

### CODE REVIEW

File: packages/smart-forms-renderer/src/utils/fhirpath.ts [hunk 1]
```diff
@@ -101,11 +101,6 @@ export function createFhirPathContext(
     rootResource: questionnaireResponse
   };
 
-  // Exit early if there are no QR items
-  if (!questionnaireResponse.item) {
```
Comment (ardihikaru) [inline packages/smart-forms-renderer/src/utils/fhirpath.ts:105]: yang ini kenapa dihilangkan ya mas?

**Review by ardihikaru — CHANGES_REQUESTED** [review]
nitip cek dikit

**Review by ardihikaru — APPROVED** [review]

File: packages/smart-forms-renderer/src/utils/fhirpath.ts (hunk 1 above)
Comment (ryuuzake) [reply packages/smart-forms-renderer/src/utils/fhirpath.ts:105]: itu perubahan dari main branch aehrc/smart-forms
bisa dilihat di commit ini
https://github.com/konsulin-id/smart-forms/pull/1/commits/d3fd1a8d37bc454b53283675ab4abcb7c2685ede

**Comment by ryuuzake** [comment]
Here are some proves that the page component works

| Storybook                                                                                 | Playground                                                                                |
|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
| image | image |
| image | image |

**Pull request merged**